sample.json

```diff
@@ -0,0 +1,15 @@
+{
+  "uuid" : "path/to/example.tif",
```

Comment:
I was under the impression that the metadata would be similarly named to and next to the image. e.g. for image `s3://somebucket/imagery/012.tif`, there would be `s3://somebucket/imagery/012.json`. Are we thinking of separating out the image and data files? If so, then I think instead of a UUID field, we would need a URI field, which was the actual URI to the image file. This would decouple the metadata from the image it is for. Otherwise, if the metadata is paired with images per a naming convention (which is scary from a maintenance perspective, one would have to be careful to move both if one is moved), then we would know the URI from the colocation/conaming.